### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ohp-inmeta**
Update readme with more info on npm packages and usage

## Describe your changes
_Replace this text with a detailed description of your changes_

## Checklist before requesting a review
- [ ] I have performed a self-review and test of my code
- [ ] If new component: Is story for component created in `stories`-folder?
- [ ] If new component: Is tsx-file import added to `packages/react/index.tsx`?
- [ ] If new component: Is css-file added to `packages/css/index.css`?

### CODE REVIEW

**Review by Jaanesen — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
-
 Består av CSS og React-komponenter for Miljødirektoratet.
 
 Her ligger det kode som implementerer noen utvalgte designelementer, henholdsvis med CSS for styling, og tilhørende React på JavaScript-siden. Koden er publisert som npm-pakker på [npm](https://www.npmjs.com/org/miljodirektoratet), noe som er nødvendig for å få versjonert og tilgjengeliggjort kode for andre utviklere på en god måte. Koden er open-source og lisensiert under MIT-lisens.
 Hvilke komponenter som er lagd ligger synlig på [Storybook-en](https://miljodir.github.io/md-components).
 
-## Kjøre opp utviklingsmiljø (storybook)
+## Npm-pakker
+Det finnes to npm-pakker som er fritt tilgjengelige fra [npm](https://www.npmjs.com/org/miljodirektoratet). Disse er:
+- [md-react](https://www.npmjs.com/package/@miljodirektoratet/md-react) for React komponentner. Kildekoden for denne pakken finnes i dette repoet, i mappen `packages/react`.
```
Comment (Jaanesen) [inline README.md:9]: "komponentner"

Comment (ohp-inmeta) [reply README.md]: Fikset

File: README.md [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+### Installasjon
+Installer pakken(e) fra npm:
 ```
-npm install
-npm run storybook
+npm install @miljodirektoratet/md-react @miljodirektoratet/md-css
 ```
 
-## Bidrag og/eller endringer
+Dersom du kun ønsker å bruke CSS (hvis du f.eks ikke bruker React i ditt prosjekt), installer du kun `@miljodirektoratet/md-css`.
```
[the remark this answers is not in the document]
Comment (ohp-inmeta) [reply README.md]: Fikset

**Pull request merged**